ckanext/dgu/bin/delete_cache_filepath.py

```diff
@@ -0,0 +1,59 @@
+'''
+Remove no longer used Resouce extra 'cache_filepath'
+'''
+
+import common
+import json
+from optparse import OptionParser
+from ckan import model
+
+from running_stats import StatsList
+
+stats = StatsList()
+
+
+class DeleteCacheFilepath(object):
+    @classmethod
+    def command(cls, config_ini, write):
+        common.load_config(config_ini)
+        common.register_translator()
+
+        rev = model.repo.new_revision()
+        rev.author = 'script-delete_cache_filepath.py'
+        all_resources = model.Session.query(model.Resource) \
+                                     .join(model.ResourceGroup)\
+                                     .join(model.Package)\
+                                     .filter(model.Package.state == 'active') \
+                                     .filter(model.Package.private == False) \
+                                     .filter(model.Resource.state == 'active')
+
+        for resource in all_resources:
+            if 'cache_filepath' in resource.extras:
+                stats.add('Removing cache_filepath', resource.id)
+                del resource.extras['cache_filepath']
```

Comment:
Think you need to save the resource again as extras are just a field on the model (unlike other places where it is a separate model).